src/main/markdown/gettingstarted.md

```diff
@@ -125,35 +160,50 @@ Change the text from "Send" to "Send to Server".
+
+The source code of the `App.java` class is displayed in the debugger. Add a breakpoint by clicking of Line number 94. Press the send button.
+The execution will stop at line 94. The stacktrace is visible, the content of the variable, etc. 
+
+<div class="screenshot"><a href="images/sdm-debugger-breakpoint.png"><img src="images/sdm-debugger-breakpoint.png" alt="Screenshot" width="42%"/></a></div>
 
 ## Compile and run in production mode<a id="compile"></a>
 
-To run the application as JavaScript in what GWT calls "production mode", compile the application by executing:
+To run the application as JavaScript in what GWT calls "production mode", create the application by executing:
 
 ```
-ant build
+mvn clean package
 ```
 
-The "build" Ant target invokes the GWT compiler which generates a number of
-JavaScript and HTML files from the MyWebApp Java source code in the
-`MyWebApp/war/` subdirectory.  To see the application, open the file
-`MyWebApp/war/MyWebApp.html` in your web browser.  The application
-should look identical to the development mode above.
+The Maven "package" goal invokes the GWT compiler which generates a number of JavaScript and HTML files from the
+MyWebApp Java source code in the `target/` subdirectory.  There you will find a `mywebapp-server-HEAD-SNAPSHOT.war`.
+You can deploy this war file to every servlet engine and run it. Once running, enter `mywebapp-server-HEAD-SNAPSHOT/index.html`
+in your web browser.  The application should look identical to the super dev mode above. You can change the name of the war using
+Maven 'buildname`.
 
 Congratulations! You've created your first web application using GWT.
 Since you've compiled the project, you're now running pure JavaScript and
-HTML that works in IE, Chrome, Firefox, Safari, and Opera. You could now deploy
-your application to production by serving the HTML and JavaScript files in your
-`MyWebApp/war/` directory from your web servers.
+HTML that works in Edge, Chrome, Firefox, Safari, and Opera. You could now deploy
+your application to production by serving the HTML and JavaScript files from your servlet engine.
+
+In case you prefer a Spring Boot server, check the 
+[gwt-maven-springboot-archetype](https://github.com/NaluKit/gwt-maven-springboot-archetype) plugin. It creates a
+project in a similar way, but uses Spring Boot instead of Jetty on the server side.
 
 ## Set up an IDE<a id="setup"></a>
 
 Now that you've created your first app, you probably want to do something a
 bit more interesting. But first, if you normally work with an IDE you'll want to
-set up Eclipse to use the GWT SDK:
+set up IntelliJ to use the GWT SDK:
 
-[Set up Eclipse](usingeclipse.html)
+[Set up IntelliJ](using-intellij.html)
 
-If you are going to stick with the command line, check out Speed Tracer     and then
+If you are going to stick with the command line, check out Speed Tracer and then
```

Comment:
I don't understand why the article talks about Speed Tracer at this point - I don't think it has been maintained since code.google.com was killed.

Reply:
yeah, you are right. remove it.